x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/nvidia/action/NvidiaActionCreatorTests.java

```diff
@@ -570,24 +569,14 @@ public void testExecute_ReturnsSuccessfulResponse_AfterTruncating_From413StatusC
         try (var sender = createSender(senderFactory)) {
             sender.startSynchronously();
 
-            var contentTooLargeErrorMessage = """
-                This model's maximum context length is 8192 tokens, however you requested 13531 tokens (13531 in your prompt;\
-                0 for the completion). Please reduce your prompt; or completion length.""";
-
-            String responseJsonContentTooLarge = Strings.format("""
+            String responseJsonContentTooLarge = """
                     {
-                        "error": {
-                            "message": "%s",
-                            "type": "content_too_large",
-                            "param": null,
-                            "code": null
-                        }
+                        "error": "Input length 18432 exceeds maximum allowed token size 8192"
```

Comment:
For this specific test case, where we're checking that a 413 status leads to truncating the input, it might be better to use an error message that doesn't match the one we check for, to confirm that it's the status code that causes the truncation rather than the error message.